spec/models/user_spec.rb

```diff
@@ -5,9 +5,54 @@
   include TestFactories
 
```

Comment:
I'll look into the difference. Using create versus new has been something I've been a little curious about since the lessons haven't been entirely clear on why one was used over the other. Are you referring to my post and comment creation in the user_with_post_and_comment factory here?